frontend/src/pages/Login.vue

```diff
@@ -127,3 +127,13 @@ export default {
   },
 }
 </script>
+<style scoped>
+.btn-gradido {
+  padding-right: 0px;
+  padding-left: 0px;
+}
+.btn-gradido-disable {
+  padding-right: 0px;
+  padding-left: 0px;
+}
```

Comment:
It seems we don't use this on login submit button
```suggestion
```

Or add disable logic & line 34:
add:
```
:variant="disabled ? 'gradido-disable' : 'gradido'"
```